packages/shared/src/components/auth/AuthenticationBanner.tsx

```diff
@@ -17,13 +17,19 @@ import {
 
 const Section = classed('div', 'flex flex-col');
 
-export function AuthenticationBanner(): ReactElement {
+type AuthenticationBannerProps = {
+  children?: ReactNode | ReactNode[];
+};
+export function AuthenticationBanner({
+  children,
+}: AuthenticationBannerProps): ReactElement {
   const { showLogin } = useAuthContext();
 
   return (
     <BottomBannerContainer
       className={classNames(
         'gap-24 border-t border-accent-cabbage-default py-10 shadow-3 laptopL:gap-32',
+        authGradientBg,
```

Comment:
re-added this for cases where people are on slow internet speeds and the BG image loads slowly